3-ecommerce-db/projects/cyf-ecommerce-api/server/components/db/.env.development

```diff
@@ -0,0 +1 @@
+DATABASE_URL = "postgres://sana:9jFvKN5g9NTnbWb0h9RwfvG77BC84dON@dpg-cmehpfun7f5s73frr2k0-a.frankfurt-postgres.render.com/exemple_postgresssql_sana"
```

Comment:
for this project better use `.env.local` file. 
check the example [here](https://github.com/UAFramework/node-1-homework/blob/feature/Denis-corrected/server/.env.local)

make sure you start npm script with this :  
`"dev": "set NODE_ENV=local && nodemon server.js",`
this must be configured in `package.json` file

make sure your local SQL server is running

make sure you have created correct database name and have it in `.env.local`